### PR TITLE
Improve table styling

### PR DESCRIPTION
**Pull request by Jondolf**
Currently, tables on news articles would look like this:

Old tables

This doesn't look great and discourages people from using tables in posts.

This PR adds styling to make tables look like this:

New tables

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

File: sass/pages/_news.scss [hunk 1]
```diff
@@ -23,6 +23,25 @@
         border-radius: $border-radius;
         max-width: 100%;
     }
+
+    table {
+        border-spacing: 0;
+        border: 2px solid $subtitle-color;
```
Comment (Jondolf) [inline sass/pages/_news.scss:29]: Using `$subtitle-color` is a bit strange here, but there isn't an existing color variable that would work

**Comment by alice-i-cecile** [comment]
Non-blocking: could we apply this to the site as a whole, rather than just the News section? I'd like to be able to use tables in the book as well, and a consistent style would be better.

**Comment by Jondolf** [comment]
Yep, I can make a `sass/elements/table.scss`. Not sure if there's some other better place

File: sass/site.scss [hunk 2]
```diff
@@ -54,4 +55,4 @@
 @import "pages/features";
 @import "pages/content";
 @import "pages/migration_guide";
-@import "pages/404";
+@import "pages/404";
```
Comment (alice-i-cecile) [inline sass/site.scss:58]:
```suggestion
@import "pages/404";

```

Comment (Jondolf) [reply sass/site.scss]: My SCSS formatter doesn't like this for some reason, I need to configure that

**Comment by TrialDragon** [comment]
Stupid question, but shouldn't this be targeting the 0.13-blog branch or whichever 0.13 branch it is based on? Maybe I'm missing something, but if this merged into main as it is, won't it also merge the WIP article in a public manner to the site?

**Comment by Jondolf** [comment]
This doesn't have the article. It was just an example to visualize what it looks like

**Comment by TrialDragon** [comment]
O, I see now. I got confused by the two different branches named quite similarly on your fork, one based on 0.13 stuff, another on main.

**Review by TrialDragon — APPROVED** [review]

**Pull request merged**